becquerel/tools/df_cache.py

```diff
@@ -85,7 +85,7 @@ def write_file(self):
             raise CacheError("Cache has not been fetched or loaded")
         try:
             self.df.to_csv(self.filename, float_format="%.12f")
-        except:
+        except Exception:
```

Comment:
More autoformatter changes? This is better than a bare `except`, but should we specialize the exception further?

Reply:
Could do but I'm not sure what the exception should be. This maintains the same behavior and is a flake8 rule